### PR TITLE
The ghost of a guess.

### PR DESCRIPTION
**Pull request by fedepedron**
I'll be merging this immediately, there was a compatibility bug between aufbau-like initial guess and ghost atoms, which have Iz = 0

### CODE REVIEW

File: lioamber/initial_guess.f90 [hunk 1]
```diff
@@ -38,13 +38,13 @@
 !##############################################################################!
 module initial_guess_data
   integer :: initial_guess = 0
-  integer :: atomic_eec(54,3)
+  integer :: atomic_eec(0:54,3)
```
Comment (ramirezfranciscof) [inline lioamber/initial_guess.f90]: Mmmm, have we become C programmers? why is this necessary?

Comment (fedepedron) [reply lioamber/initial_guess.f90]: Nothing to do with C. The thing is, ghost atoms have Iz=0; this is done this way to avoid checking "if Iz .gt. 0" later during the loops in the aufbau-like initial guess.
